__tests__/lib/services/analysisService.test.ts

```
@@ -0,0 +1,66 @@
+import { httpClient } from "@/lib/api";
+import { CREATE_ANALYSIS, GET_ANALYSIS_BY_ID, UPDATE_ANALYSIS } from "@/lib/constants/api/analysis";
+import { BASE_URL } from "@/lib/constants/config";
+import { getAnalysisById, createAnalysis, renameAnalysis } from "@/lib/services/analysisService";
+
+jest.mock("@/lib/api", () => ({
+  httpClient: { get: jest.fn(), post: jest.fn(), patch: jest.fn() },
+}));
+
+const mockHttpClient = httpClient as jest.Mocked<typeof httpClient>;
+const mockResponse = (data: any, status = 200) => ({ data, status, statusText: "OK", headers: {} });
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)